packages/labs/ssr/README.md

```diff
@@ -73,8 +73,8 @@ context.body = Readable.from(ssrResult);
 
 ```js
 import {myTemplate} from './my-template.js';
-import {render, hydrate} from `lit/experimental-hydrate.js`;
-
+import {render} from `lit-html`;
```

Comment:
Why not from `lit`?

Reply:
No reason, I think? I'm guessing there was a timing thing with when we added lit and that just didn't get updated? I don't think there's a reason.